### DISCUSSION TITLE
Make default `routeTree.gen.ts` file more auto-descriptive?

### DISCUSSION THREAD

**Discussion by swierzbicki** [TanStack/router#2052] [Ideas]
Is it possible to change default file name to more explicit and auto-descriptive? 

For example:
- `routeTree.generated.ts` 
- `routeTree.autogenerated.ts` 

It is not so obvious that `gen` means `generate...` 

So it would be always easier for people who are not familiar with the TanStack Router library.

**Comment by BrendanC23** [comment]
You can change the defaults: https://tanstack.com/router/latest/docs/framework/react/guide/file-based-routing#configuration. See the `generatedRouteTree` option.

**Reply by swierzbicki** [reply]
@BrendanC23 I am aware of that 😉 

This is only an idea to change the default to be more self-explanatory/explicit. Worth to consider 😉
